enclave/tls_cert.c

```diff
@@ -0,0 +1,215 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// clang-format off
```

Comment:
Why clang format needs to be turned off?

Reply:
It was added from earlier prototype work, it;s not needed anymore.

Reply:
The header reordering from running clang formatting causes compiler errors on the cert related type defines.